tests/console/commands/test_build.py

```diff
@@ -16,11 +18,15 @@
     from tests.types import FixtureDirGetter
 
 
-def test_build_with_multiple_readme_files(
+@pytest.mark.parametrize(
+    "output_dir", [None, "dist", "test/dir", "../dist", "absolute"]
+)
+def test_build_with_multiple_readme_files_and_output_option(
```

Comment:
I'd prefer to leave this test alone and add a new test. You could use `test_build_creates_packages_in_dist_directory_if_no_output_is_specified` from #8877 as starting point (and copy the relevant stuff from there).

Reply:
@radoering done!